.github/workflows/security_owasp.yml

```diff
@@ -0,0 +1,13 @@
+name: Security OWASP dependency check
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "43 8 * * MON-FRI" # Every weekday at 08:43 UTC
+jobs:
+  security-kotlin-owasp-check:
+    name: Kotlin security OWASP dependency check
+    uses: ministryofjustice/hmpps-github-actions/.github/workflows/security_owasp.yml@v2 # WORKFLOW_VERSION
+    with:
+      channel_id: ${{ vars.SECURITY_ALERTS_SLACK_CHANNEL_ID || 'NO_SLACK' }}
+      subproject: hmpps-kotlin-spring-boot-autoconfigure
```

Comment:
note only running the security check for the autoconfigure. We previously ran for whole project but that was unnecessary and caused failures for the test-apps that we don't care about